site/smart-accounts/accounts/light-account.md

```diff
@@ -20,33 +20,55 @@ head:
+- **Security:** Light Account has undergone a thorough audit by Quantstamp to ensure its safety and reliability.
+- **Compatibility:** It is fully compatible with ERC-1271 for signature support, making it portable for various applications, such as those on OpenSea.
+
+## Future-Proofed with Modular 6900 Accounts
+
+Light Account is forward-compatible and upgradeable for future, more optimized versions and the upcoming modular 6900 account standard, ensuring you'll stay on the cutting edge of EVM accounts.
+
+At Alchemy, we prioritize fundamental principles and community collaboration. Your trust and safety are paramount in the new digital frontier. Choose Light Account for an optimized, open, and secure smart account solution.
 
 ## Using Light Account
 
-The code snippet below demonstrates the usage of Light Account with Account Kit. It creates a Light Account and sends a `UserOperation` from it:
+The code snippet below demonstrates how to use Light Account with Account Kit. It creates a Light Account and sends a `UserOperation` from it:
 
 <!--@include: ../../getting-started.md{56,68}-->
 
+## Details
+
+Light Account is [audited by quantstamp](https://github.com/alchemyplatform/light-account/blob/main/Quantstamp-Audit.pdf), is gas-optimized, and compatible as a [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart account implementation.
+
+Based on Ethereum Foundation’s canonical [SimpleAccount](https://github.com/eth-infinitism/account-abstraction/blob/develop/contracts/samples/SimpleAccount.sol) with key improvements for production app developers, Light Account:
+
+- significantly reduce gas costs
+- [ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) has signature support to ensure users can sign messages, such as on OpenSea
+- allows ownership transfer so that users won’t get locked into a single `Signer`
+
 ## Benchmarks
 
-Here are some benchmarks for Light Account and other smart account implementations you might consider:
-
-| Account              | Creation | Native transfer | ERC20 transfer | Total Gas |
-| -------------------- | -------- | --------------- | -------------- | --------- |
-| Alchemy LightAccount | 279746   | 100844          | 90345          | 470935    |
-| Kernel v2.1-lite     | 230968   | 101002          | 90321          | 422291    |
-| Kernel v2.1          | 265215   | 106460          | 96038          | 467713    |
-| Biconomy             | 270013   | 104408          | 93730          | 468151    |
-| Etherspot            | 279219   | 103719          | 93324          | 476262    |
-| Kernel v2.0          | 339882   | 110018          | 99622          | 549522    |
-| SimpleAccount        | 383218   | 101319          | 90907          | 575444    |
+Light Account was built to optimize the most common user flows, providing the lowest gas cost for transfers. Here is a table of Light Account benchmarked against other smart account implementations:
```

Comment:
We can probably shorten this.

"Light Account optimizes for transfers (the most common user flow). Below is a table of Light Account benchmark against other smart account implementations:"